molecule_ec2/cookiecutter/{{cookiecutter.molecule_directory}}/{{cookiecutter.scenario_name}}/create.yml

```diff
@@ -73,53 +73,49 @@
         filters:
           name: "{{ item.image_name }}"
       loop: "{{ molecule_yml.platforms }}"
-      when: item.image is not defined
+      when: item.image_id is not defined
       register: ami_info
 
     - name: Create molecule instance(s)
-      ec2:
+      ec2_instance:
         key_name: "{{ keypair_name }}"
-        image: "{{ item.image
-          if item.image is defined
+        image_id: "{{ item.image_id
+          if item.image_id is defined
           else (ami_info.results[index].images | sort(attribute='creation_date', reverse=True))[0].image_id }}"
         instance_type: "{{ item.instance_type }}"
         vpc_subnet_id: "{{ item.vpc_subnet_id }}"
-        group: "{{ security_group_name }}"
-        instance_tags: "{{ item.instance_tags | combine({'instance': item.name})
+        security_group: "{{ security_group_name }}"
+        region: "{{ item.region
+          if item.region is defined
+          else omit }}"
+        tags: "{{ item.instance_tags | combine({'Name': item.name})
           if item.instance_tags is defined
-          else {'instance': item.name} }}"
+          else {'Name': item.name} }}"
+        filters:
+          instance-state-name: running
+          subnet-id: "{{ item.vpc_subnet_id }}"
+          tag:Name: "{{ item.name }}"
         wait: true
-        assign_public_ip: true
-        exact_count: 1
-        count_tag:
-          instance: "{{ item.name }}"
+        wait_timeout: 300
+        network:
+          assign_public_ip: true
       register: server
       loop: "{{ molecule_yml.platforms }}"
       loop_control:
         index_var: index
-      async: 7200
-      poll: 0
-
-    - name: Wait for instance(s) creation to complete
```

Comment:
Removing the creation waits is a substantial change, this shifts the burden of waiting for the creation to the prepare file.

To me this is very non-obvious, at the end of the create step the instance should have been created.
At the very least, this change should have accompanying documentation updates.

I know the initial proposal used cloud_init_data_facts as a replacement wait process.
That suffers from being Linux specific, it doesn't support Windows or FreeBSD based systems.